extensions/wikia/Wall/maintenance/CleanupCommentsIndex.php

```diff
@@ -0,0 +1,26 @@
+<?php
+require_once( getenv( 'MW_INSTALL_PATH' ) !== false ? getenv( 'MW_INSTALL_PATH' ) . '/maintenance/Maintenance.php' : dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
```

Comment:
I'd simply use `__DIR__` and go up the directory tree to the `maintenance` directory